stories/Navbar/NavbarInlineExample.js

```diff
@@ -1,112 +1,108 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Collapse, Icon, LinkList, LinkListItem } from '../../src'
 
-class NavbarInlineExample extends React.Component {
-  defaultState = {
-    collapseOpen1: false,
-    collapseOpen2: false,
-    collapseOpen3: false
-  }
+const NavbarInlineExample = () => {
+  const [collapseOpen1, setCollapseOpen1] = useState(false)
+  const [collapseOpen2, setCollapseOpen2] = useState(false)
+  const [collapseOpen3, setCollapseOpen3] = useState(false)
 
-  state = {
-    ...this.defaultState
+  const expanded = {
+    'aria-expanded': true
   }
-
-  toggle = (e, id) => {
+  const toggle = (e, id) => {
     e.preventDefault()
-    this.setState({
-      [`collapseOpen${id}`]: !this.state[`collapseOpen${id}`]
-    })
-  }
-
-  render() {
-    const { collapseOpen1, collapseOpen2, collapseOpen3 } = this.state
-
-    const expanded = {
-      'aria-expanded': true
+    switch (id) {
+      case 1:
+        setCollapseOpen1(!collapseOpen1)
+        break
```

Comment:
nitpick: `return setCollapseOpen1(!collapseOpen1)` and save a line each :)

Reply:
Better coding practices 💯